src/containers/Root/UnauthorizedRoot.tsx

```diff
@@ -1,7 +1,25 @@
 import React from 'react';
+import { Route, Switch } from "react-router-dom";
+
+import { styled } from '../../styles';
+
+import { Greeting } from "../Login";
+
+const Wrapper = styled.div`
+  width: 100vw;
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: ${({theme})=> theme.colors.backgroundGray};
+`;
 
 export const UnauthorizedRoot: React.FC = () => {
   return (
-    <></>
+    <Wrapper>
+      <Switch>
+        <Route path={'/'} render={ () => <Greeting /> } />
```

Comment:
i guess, would be better to use component instead of render because we will not pass any props